network/wsNetwork.go

```diff
@@ -417,8 +417,9 @@ type WebsocketNetwork struct {
 	// messagesOfInterestMu protects messagesOfInterest and ensures
 	// that messagesOfInterestEnc does not change once it is set during
 	// network start.
-	messagesOfInterestMu   deadlock.Mutex
-	messagesOfInterestCond *sync.Cond
```

Comment:
I had to do the same thing in the participation registry. Channels are a lot easier to reason about.